WcaOnRails/spec/controllers/competitions_controller_spec.rb

```diff
@@ -463,20 +463,22 @@
   end
 
   describe 'GET #post_announcement' do
+    let(:delegate_team_member) { FactoryGirl.create(:team_member, :senior_delegate, start_date: "2010-01-01") }
+    let(:competition_to_post) { FactoryGirl.create(:competition, delegates: [delegate_team_member.user]) }
```

Comment:
You can use `FactoryGirl.create(:delegate, start_date: ...)` in the first line and omit `user` in the second one.